forumhelpers/FHP.js

```diff
@@ -66,8 +68,33 @@ function getCount(name) {
 
 
 //Search for a user
-var searchBar = document.getElementById("searchBar")
-var searchButton = document.getElementById("searchButton")
+var searchBar = document.getElementById("searchBar");
+var searchButton = document.getElementById("searchButton");
+
+var searchOptions = [];
+function updateUserChoice() {
+	document.getElementById("searchOptions").innerHTML = "";
+	searchOptions = [];
+	var searchedLetters = searchBar.value
+	for (k = 0; k < membersList.length; k++) {
+		if (membersList[k].toUpperCase().includes(searchedLetters.toUpperCase()) && searchOptions.length != 5 && searchedLetters != "") {
+			searchOptions.push(membersList[k]);
+		}
+	}
+	for (k = 0; k < searchOptions.length; k++) {
+		var searchOptionBox = document.createElement("DIV");
+		searchOptionBox.innerText = searchOptions[k];
+		searchOptionBox.setAttribute("class", "searchOption");
+		searchOptionBox.setAttribute("onclick", `autoFill("${searchOptions[k]}")`);
+		document.getElementById("searchOptions").appendChild(searchOptionBox);
+	}
+}
+
+function autoFill(fillValue) {
+	searchBar.value = fillValue;
+	checkUser();
+	updateUserChoice();
+}
```

Comment:
On click of the autofill value hopefully should automatically scroll down to the user.

Reply:
Skimming the code shows that _maybe_ you already intended to do that, but it's not working for me. Otherwise, great PR! :heart:

Reply:
Good idea! It should be as easy as running searchUser() under updateUserChoice() in the code you selected above.